plugin/ale/cursor.vim

```diff
@@ -7,6 +7,20 @@ endif
 
 let g:loaded_ale_cursor = 1
 
+" Return a formatted message according to g:ale_echo_msg_format variable
+function! s:GetMessage(linter, type, text) abort
+    let msg = g:ale_echo_msg_format
+    let linter = a:linter
+    let type = a:type ==# 'E' ? 'Error' : 'Warning'
```

Comment:
I wonder if the severity strings should be configurable too. That might be handy for supporting other languages, or abbreviations, etc.